### DISCUSSION TITLE
Inquiry About Live Streaming Transcription in Microsoft Teams

### DISCUSSION THREAD

**Discussion by karthickraja-shanmugaguru** [deepgram/community#1025] [General help] [answered]
We have successfully implemented live streaming transcription in Zoom, following your guidance, and it has been working effectively for our requirements. Thank you for your support in enabling this integration.

We are now exploring the possibility of implementing a similar live streaming transcription setup in Microsoft Teams. Could you please confirm:

1. If it is possible to integrate live streaming transcription with Microsoft Teams using ngrok?
2. Whether the speed and quality of transcription in Microsoft Teams would be comparable to our experience with Zoom?

**Comment by deepgram-community** [comment]
Thanks for asking your question. Please be sure to reply with as much detail as possible so the community can assist you efficiently.
<sub>Consider joining our [Discord community](https://discord.gg/deepgram) for more opportunity to engage with your fellow Deepgram users. You can earn points which can be redeemed for cool stuff by being active in our communities!</sub>

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment, marked as answer]
I'm not aware of anyone who's done that integration with MS Teams yet nor am super familiar with their API / Platform. You'd have to do some research based on what Microsoft Provides in terms of API support.  In theory you could connect a service like Deepgram to MS Teams, but it all depends on what they offer you in terms of API functionality in order to get access to the audio created from a meeting.

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
